.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
we should prob make the settings here closer to the eslint settings so that there aren't so many changes in this PR
https://prettier.io/docs/en/configuration.html